content/building/yaml.md

```diff
@@ -23,18 +23,17 @@ The YAML feature is currently in *beta* and has the following limitations:
 You can get started with YAML easily if you have an existing project set up on Codemagic. 
 
 1. Navigate to your app settings.
-2. Expand the **Advanced configuration (beta)** tab.
-3. Click **Download configuration** and save the generated `codemagic.yaml` file to a suitable location. 
+2. Click **Download configuration** on the right sidebar in the **Configuration as code** section.
 
-Note that in order to use the file for build configuration on Codemagic, it has to be committed to your repository. The name of the file must be `codemagic.yaml` and it must be located in the root directory of the project. 
+Note that in order to use the file for build configuration on Codemagic, it has to be committed to your repository. The name of the file must be `codemagic.yaml` and it must be located in the root directory of repository.
 
 ## Encrypting sensitive data
 
 During the export, Codemagic automatically encrypts the secret environment variables in your build configuration. 
 
 If you wish to add new environment variables to the YAML file, you can encrypt them via Codemagic UI. 
 
-1. In your app settings > Advanced configuration (beta), click **Encrypt environment variables**.
+1. In your app settings on the right sidebar in the **Configuration as code** section, click **Encrypt environment variables**.
```

Comment:
this on the right sidebar in the...  @helinanever sounds a bit rocky, maybe you have some rename ideas? or just put
app settings > right sidebar > configuration as code...?

Reply:
Changed in a way it's written before https://github.com/codemagic-ci-cd/codemagic-docs/pull/55/commits/ed0d1ed35f16aae887ffe06b553b1a2d22704fc7